pom.xml

```diff
@@ -419,44 +328,6 @@
         <plugins>
```

Comment:
The whole fluo-io-release profile can be deleted. None of it should be necessary since all of it is contained in ASF parent pom.